auth_server/authz/acl.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/cesanta/docker_auth/auth_server/authn"
 	"github.com/cesanta/glog"
+	"github.com/schwarmco/go-cartesian-product"
```

Comment:
this needs to be vendored

Reply:
I have already added it to the vendor.json. Is there something else that needs to be done to vendor it?